### ISSUE TITLE
Is provided Firecamp as VSCODE extension ?

### ISSUE THREAD

**Issue by clabnet** [firecamp-dev/firecamp#169]
This code seems very powerfull.
Have you provided to create it also as VSCODE extension ? 
Thank you

**Comment by burkaslarry** [comment]
anyone?

**Comment by Hussseinkizz** [comment]
Yeah I think this should have a vscode extension, who is up we take on the challenge?

**Comment by Nishchit14** [comment]
Thanks to everyone for sharing your interest in the Firecamp VSCode Extension. To be honest, as of now, I don't have any plans to build it. I am currently busy with building other important features like Doc and Runner. I am leaving this issue open to discuss the various thoughts and to measure community interest for VSCode Ext.

**Comment by Hussseinkizz** [comment]
Yeah I know priorities now but it's really needed and would give this a boost you can't imagine, just like it did for thunder client, a vscode extension would really be very great and more convenient, so anyone with any knowledge of how we can build one, am willing to collaborate we see it happen!
